### DISCUSSION TITLE
Can't fabricate items that require a talent in Outpost Deathmatch

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#14861] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/a1e60eb8-f29c-4a77-ac90-134eba7cbac3)
no crafting allowed!!!!!!!!!!!!!!!!!!!!!

### Reproduction steps

1. load in PVP mode
2. try to craft something cool

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I'm not sure if I understood the issue correctly. Do you mean that you can't fabricate anything, or that you can't fabricate items that are unlocked by a talent? I'm afraid I could not spot any issues with the fabrication in PvP: I seem to be able to fabricate both just fine (assuming I have the appropriate talent ofc):

![image](https://github.com/user-attachments/assets/35bf362f-aadc-4733-94d2-df716c03eccb)

**Reply by girlyguppy** [reply]
Yeah, I meant specifically the unlockable recipes
You can't get talents in pvp though, so it's locked

You can't even use medals to give yourself xp, last I checked, they did nothing 

Meaning the only way is cheats or mods

**Reply by Regalis11** [reply]
> You can't get talents in pvp though, so it's locked

Yes you can? You can select talents as perks.

**Reply by girlyguppy** [reply]
> > You can't get talents in pvp though, so it's locked
> 
> Yes you can? You can select talents as perks.

Perks aren't allowed in outpost deathmatch

**Reply by Regalis11** [reply]
Ah, I didn't realize the issue was about the outpost deathmatch specifically. I suppose we could consider ways to allow fabricating these restriced items there too, even though I feel it might somewhat go against the way the deathmatch mode is intended to play out (with people getting weapons mainly through the weapon drops, as opposed to crafting). Being able to fabricate everything would for sure screw up the balance at least - many of the lower-tier weapons would become useless and the new meta might be to just fabricate the best gear immediately and use nothing but those.

**Reply by Crazyjr20** [reply]
I think having the option is better than not, as no vanilla maps have materials aside from deconning, which is a slow thing. Even then you could disable interactions with fabricators or have none at all to avoid it as a map maker.
Though to be honest I don't imagine we will see many people using fabricators even with recipes in this mode.

**Reply by girlyguppy** [reply]
> Ah, I didn't realize the issue was about the outpost deathmatch specifically. I suppose we could consider ways to allow fabricating these restriced items there too, even though I feel it might somewhat go against the way the deathmatch mode is intended to play out (with people getting weapons mainly through the weapon drops, as opposed to crafting). Being able to fabricate everything would for sure screw up the balance at least - many of the lower-tier weapons would become useless and the new meta might be to just fabricate the best gear immediately and use nothing but those.

Crates should drop more often and symmetrically then, because as it currently is, half the round is 500 revolvers vs 1 guy with an autoshotgun who hoarded 5 crates worth of ammo...

Having the option at least to unlock recipes is good
I wish crate spawns were up to the mapmaker, because they like to clump and spawn infrequently

I've hosted pvp a lot recently, and I made a pvp outpost for testing, and everyone seems to agree crates are too infrequent
